easybuild/easyconfigs/s/SCOTCH/SCOTCH-7.0.3-gompi-2022b.eb

```
@@ -0,0 +1,56 @@
+easyblock = 'CMakeMake'
+
+name = 'SCOTCH'
+version = '7.0.3'
+
+homepage = 'https://www.labri.fr/perso/pelegrin/scotch/'
+description = """Software package and libraries for sequential and parallel graph partitioning,
+static mapping, and sparse matrix block ordering, and sequential mesh and hypergraph partitioning."""
+
+toolchain = {'name': 'gompi', 'version': '2022b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://gitlab.inria.fr/scotch/scotch/-/archive/v%(version)s/']
+sources = ['%(namelower)s-v%(version)s.tar.gz']
+checksums = ['5b5351f0ffd6fcae9ae7eafeccaa5a25602845b9ffd1afb104db932dd4d4f3c5']
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+    ('Bison', '3.8.2'),
+    ('flex', '2.6.4'),
+]
+
+dependencies = [
+    ('zlib', '1.2.12'),
+]
+
+configopts = '-DBUILD_SHARED_LIBS=ON '
+configopts += '-DINSTALL_METIS_HEADERS=OFF '
+configopts += '-DCMAKE_BUILD_TYPE=Release '
```

shared libs, and targetting `Release` build type, are default behaviour in the `CMakeMake` easyblock
```suggestion
configopts += '-DINSTALL_METIS_HEADERS=OFF '
```